### PR TITLE
[[ Bug 21297 ]] Fix issue with importing foreign value types that require conversion

### PR DESCRIPTION
**Pull request by livecodeian**

### CODE REVIEW

File: docs/notes/bugfix-21297.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# LCB: Fix issue with importing foreign value types that require conversion
```
Comment (montegoulding) [inline docs/notes/bugfix-21297.md]: @livecodeian I think this note needs to be in `docs/lcb/notes`

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 07f09f1

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 07f09f1638f4a53e8186be1f56905a494b6ae42d

**Comment by livecode-vulcan** [comment]
😎 test success 07f09f1638f4a53e8186be1f56905a494b6ae42d

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/1471)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1487)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1461)
* try-community-universal-ios-iphoneos11.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.3/builds/64)
* try-community-universal-ios-iphonesimulator11.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.3/builds/64)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/1465)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1491)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/1471)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1485)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/1470)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1480)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1520)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1460)

**Pull request merged**